### DISCUSSION TITLE
Inference Score for Intel Xeon E-2176G

### DISCUSSION THREAD

**Discussion by Iceman248** [blakeblackshear/frigate#2133] [Ask A Question] [answered]
I started at 34-35ms using Docker on unRAID, then after getting hardware acceleration working on the onboard P630 iGPU, it is now 32ms. I just wanted to ask since I couldn't find if I am getting the performance I should be? I have Coral on order, but in the meantime trying to make sure I get best use out of what I have. Thanks.

**Comment by blakeblackshear** [comment, marked as answer]
Those are actually very good speeds for CPU detectors.
